### ISSUE TITLE
Cap the max size of auto-expanded ListMorph result bubbles.

### ISSUE THREAD

**Issue by cycomachead** [jmoenig/Snap--Build-Your-Own-Blocks#1110]
Look at this... 
<img width="1439" alt="screen shot 2016-01-13 at 3 13 51 pm" src="https://cloud.githubusercontent.com/assets/1505907/12310950/7a76353a-ba08-11e5-8178-c7573d399a85.png">

**Comment by jmoenig** [comment]
yeah, so? This is what everybody was rooting for! You can always make it smaller manually, same as you could make it bigger before...

**Comment by cycomachead** [comment]
Taking up the _entire_ scripting area looks wrong, and it's not clear how to close it. Resizing just to close things seems weird, but after a couple tries I realized it actually works to click anywhere outside the scripting area, though I honestly never noticed that before.

But there's a difference between "bigger" and "engulfing your workspace".

**Comment by jmoenig** [comment]
just clicking *anywhere* outside of the speech bubble closes it, it doesn't have to be inside the scripting area. Yeah, the max is the dimensions of the scripting area. That's the *point*. If you want a smaller version, make a stage watcher. But for checking the contents of a big list, this is *exactly* what you want. I don't get how this is an inconvenience.
